README.md

```diff
@@ -1,3 +1,15 @@
+![Leanplum](Leanplum.png)
+
+<p align="center">
+<img src='https://jenkins.leanplum.com/buildStatus/icon?job=apple-sdk-github'></a>
```

Comment:
can you please add width & height attributes to the image tags, so it doesn't jump on page load. also adding alt tag could be helpful